src/Tizen.NUI.Components/Controls/RecyclerView/LayoutManager.cs

```diff
@@ -0,0 +1,127 @@
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum Orientation
+        {
+            /// <summary>
+            /// Vertical
+            /// </summary>
+            /// <since_tizen> 8 </since_tizen>
+            /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Vertical = 0,
+            /// <summary>
+            /// Horizontal
+            /// </summary>
+            /// <since_tizen> 8 </since_tizen>
+            /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Horizontal = 1,
+        }
+
+        /// <summary>
+        /// Container which contains RecycleItems.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View Container{get;set;}
+ 
+        /// <summary>
+        /// Size of RecycleItem.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public Size ItemSize{get;set;} = new Size();
+ 
+        /// <summary>
+        /// Get/Set the orientation in the layout.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public Orientation LayoutOrientation{get;set;} = Orientation.Vertical;
+
+        /// <summary>
+        /// How far can you reach the next item.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public float StepSize{
+            get
+            {
+                return mStepSize;
+            }
+        }
+
+        /// <summary>
+        /// This is called to find out where items are lain out according to current scroll position.
+        /// </summary>
+        /// <param name="scrollPosition">Scroll position which is calculated by ScrollableBase</param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        public virtual void Layout(float scrollPosition)
+        {
+           
+        }
+
+        /// <summary>
+        /// This is called to find out which items should be recycled according to current scroll position.
+        /// </summary>
+        /// <param name="scrollPosition">Scroll position which is calculated by ScrollableBase</param>
```

Comment:
scrollPosition always means that the position of scrolling container in ScrollableBase.

In many case, top left coordination but it can be changed because components can use other pivot point and parent instead of top-left